### PR TITLE
Fix 55 warnings related to flyouts.

### PR DESCRIPTION
**Pull request by samelhusseini**
## The basics

- [x] I branched from develop
- [x] My pull request is against develop
- [x] My code follows the [style guide](https://developers.google.com/blockly/guides/modify/web/style-guide)

### Resolves

Fix 55 warnings

### Proposed Changes

Everything's a little intertwined so it was hard to split.

### Test Coverage

Tested on:

### CODE REVIEW

File: core/flyout_base.js [hunk 1]
```diff
@@ -431,7 +434,7 @@ Blockly.Flyout.prototype.hide = function() {
 
 /**
  * Show and populate the flyout.
- * @param {!Array|string} xmlList List of blocks to show.
```
Comment (rachel-fenichel) [inline core/flyout_base.js]: Is the nodeList nullable?

File: core/workspace_svg.js [hunk 2]
```diff
@@ -2351,6 +2371,17 @@ Blockly.WorkspaceSvg.setTopLevelWorkspaceMetrics_ = function(xyRatio) {
...
 };
 
+/**
+ * Finds the top-level blocks and returns them.  Blocks are optionally sorted
+ * by position; top to bottom (with slight LTR or RTL bias).
+ * @param {boolean} ordered Sort the list if true.
+ * @return {!Array.<!Blockly.BlockSvg>} The top-level block objects.
+ * @override
+ */
+Blockly.WorkspaceSvg.prototype.getTopBlocks = function(ordered) {
```
Comment (rachel-fenichel) [inline core/workspace_svg.js]: This is just changing the return type, right?

**Comment by rachel-fenichel** [comment]
Per offline discussion, it seems better to just cast in getSvgRoot and getInjectionDiv, since all uses are required anyway.

File: core/workspace_svg.js (hunk 2 above)
Comment (samelhusseini) [reply core/workspace_svg.js]: Correct.

File: core/trashcan.js [hunk 3]
```diff
@@ -49,6 +49,12 @@ Blockly.Trashcan = function(workspace) {
    */
   this.contents_ = [];
 
+  /**
+   * The trashcan flyout.
+   * @type {Blockly.Flyout}
+   * @private
+   */
+  this.flyout_ = null;
```
[the remark this answers is not in the document]
Comment (samelhusseini) [reply core/trashcan.js]: Make sense, I'll update.

File: core/flyout_base.js (hunk 1 above)
Comment (samelhusseini) [reply core/flyout_base.js]: Nice, good catch.

File: core/workspace_svg.js [hunk 4]
```diff
@@ -564,6 +571,8 @@ Blockly.WorkspaceSvg.prototype.getOriginOffsetInPixels = function() {
 /**
  * Return the injection div that is a parent of this workspace.
  * Walks the DOM the first time it's called, then returns a cached value.
+ * Note: We assume this is only called after the workspace has been injectioned
```
Comment (rachel-fenichel) [inline core/workspace_svg.js]: Nit: injectioned -> injected

File: core/workspace_svg.js [hunk 5]
```diff
@@ -934,21 +944,22 @@ Blockly.WorkspaceSvg.prototype.getBubbleCanvas = function() {
 
 /**
  * Get the SVG element that contains this workspace.
- * @return {SVGElement} SVG element.
+ * Note: We assume this is only called after the workspace has been injectioned
```
Comment (rachel-fenichel) [inline core/workspace_svg.js]: Nit: injectioned -> injected

**Pull request merged**